src/js/PluginActions.js

```diff
@@ -0,0 +1,2 @@
+var {PluginActions} = window["marathonPluginInterface"];
```

Comment:
Why are you duplicating this into a new file instead of just using it from the marathonPluginInterface directly?